LICENSE

```diff
@@ -1,4 +1,4 @@
-   Copyright © 2020 Progress Software Corporation and/or one
+   Copyright © 2021 Progress Software Corporation and/or one
```

Comment:
Don't remove 2020. Change this to be 2020-2021 or 2020, 2021. The copyright needs to cover each year that there were changes

Reply:
Misunderstood the purpose of this file and the years. Fixed.